Tribler/Test/Core/Modules/RestApi/test_debug_endpoint.py

```diff
@@ -31,19 +30,18 @@ def verify_response(response):
             response_json = json.twisted_loads(response)
             self.assertEqual(len(response_json["slots"]["random"]), 4)
 
-        self.should_check_equality = False
```

Comment:
Why were these removed?

Reply:
Because these are unnecessary. If `should_check_equality` is `True`, the result of the request will be checked against `expected_json` argument, which is None by default. I can't imagine a situation where the code would expect the returned JSON value to really be `None`. On the other hand, I can't imagine a situation in which one would set `expected_json` to something other than `None`, and will not set `should_check_equality`, either.

Therefore, it is perfectly fine to consider that `should_check_equality` is `False` in all cases where `expected_json` is `None`, and it is True if `expected_json` is not `None`.
And that is fewer code lines and less mental burden for the programmer.